sql/_hny_join.sql

```diff
@@ -153,44 +185,120 @@ WITH
+            SUM(h.middle_income_units::NUMERIC) as middle_income_units,
+            SUM(h.other_income_units::NUMERIC) as other_income_units,
+            SUM(h.studio_units::NUMERIC) as studio_units,
+            SUM(h."1_br_units"::NUMERIC) as "1_br_units",
+            SUM(h."2_br_units"::NUMERIC) as "2_br_units",
+            SUM(h."3_br_units"::NUMERIC) as "3_br_units",
+            SUM(h."4_br_units"::NUMERIC) as "4_br_units",
+            SUM(h."5_br_units"::NUMERIC) as "5_br_units",
+            SUM(h."6_br+_units"::NUMERIC) as "6_br+_units",
+            SUM(h.unknown_br_units::NUMERIC) as unknown_br_units,
+            SUM(h.counted_rental_units::NUMERIC) as counted_rental_units,
+            SUM(h.counted_homeownership_units::NUMERIC) as counted_homeownership_units
+                            
+        FROM RELATEFLAGS_hny_matches r
+        LEFT JOIN HNY_geo h
+        ON r.hny_id = h.hny_id
+        WHERE r.one_dev_to_many_hny = 1 AND r.one_hny_to_many_dev = 1
+        GROUP BY r.job_number, r.one_dev_to_many_hny, r.one_hny_to_many_dev),
+    many_to_many as (
+        SELECT
+            hny_id,
+            MIN(job_number) as job_number, 
+            MAX(classa_hnyaff) as classa_hnyaff,
+            MAX(all_hny_units) as all_hny_units,
+            one_dev_to_many_hny,
+            one_hny_to_many_dev,
+            MIN(project_start_date) as project_start_date,
+            MIN(project_completion_date) as project_completion_date,
+            MAX(extremely_low_income_units::NUMERIC) as extremely_low_income_units,
+            MAX(very_low_income_units::NUMERIC) as very_low_income_units,
+            MAX(low_income_units::NUMERIC) as low_income_units,
+            MAX(moderate_income_units::NUMERIC) as moderate_income_units,
+            MAX(middle_income_units::NUMERIC) as middle_income_units,
+            MAX(other_income_units::NUMERIC) as other_income_units,
+            MAX(studio_units::NUMERIC) as studio_units,
+            MAX("1_br_units"::NUMERIC) as "1_br_units",
+            MAX("2_br_units"::NUMERIC) as "2_br_units",
+            MAX("3_br_units"::NUMERIC) as "3_br_units",
+            MAX("4_br_units"::NUMERIC) as "4_br_units",
+            MAX("5_br_units"::NUMERIC) as "5_br_units",
+            MAX("6_br+_units"::NUMERIC) as "6_br+_units",
+            MAX(unknown_br_units::NUMERIC) as unknown_br_units,
+            MAX(counted_rental_units::NUMERIC) as counted_rental_units,
+            MAX(counted_homeownership_units::NUMERIC) as counted_homeownership_units
+        FROM _many_to_many
+        GROUP BY hny_id,  one_dev_to_many_hny, one_hny_to_many_dev
+    ) 
+-- 6) Insert into HNY_devdb_lookup  
 SELECT 
 *
 INTO 
-HNY_devdb
-FROM (SELECT * FROM one_to_many
-    UNION
-    SELECT * FROM other_relations) all_hny;
+HNY_devdb_lookup
+FROM (
+SELECT * FROM one_to_one
+UNION
+SELECT * FROM one_to_many
+UNION
+SELECT * FROM many_to_one
+UNION
+SELECT * FROM many_to_many) all_hny;
```

Comment:
Why do we give the nested query an alias? Not against it just curious. Think it's nice for clarity and it can be merged in and curious about your thoughts here

Reply:
I think it is a sql or postgres requirement and it would throw error when I don't use it